libplatsupport/plat_include/rockpro64/platsupport/plat/timer.h

```diff
@@ -1,67 +1,8 @@
 /*
- * Copyright 2019, Data61, CSIRO (ABN 41 687 119 230)
+ * Copyright 2025, UNSW
```

Comment:
You can't just remove the original copyright, except if it was wrong for some reason (but better have a good commit message then).